loadable_apps/loadable_sample/wifiapp/wifiapp.c

```diff
@@ -70,6 +74,10 @@ int wifiapp_main(int argc, char **argv)
 
 	printf("This is WIFI App\n");
 
+#ifdef CONFIG_WIFI_PLATFORM_LIB
+	wifi_pf_test();
```

Comment:
How about moving this before line 75?
Because normally platform code initialization is called before application code.

Reply:
Done.